ui/src/containers/Drawer/components/WheelComponents.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import { Circle, Shape } from 'react-konva'
+
+const COUNT = 8
+const START_DEG = -0.7
+const DEG = (2 / (COUNT * 2)) * Math.PI
+
+const MARGIN = 20
+const NEEDLE_HEIGHT = 50
+const PADDING = NEEDLE_HEIGHT + MARGIN
+const RADIUS = 150
+const WHEEL_CENTER_RADIUS = 40
+const WHEEL_OUTLINE_WIDTH = 15
+const WHEEL_OUTLINE_BORDER_WIDTH = WHEEL_CENTER_RADIUS / 10
+const WHEEL_CENTER = {
+  X: PADDING + RADIUS,
+  Y: PADDING + RADIUS
+}
+const NEEDLE_WIDTH = WHEEL_CENTER_RADIUS
+const NEEDLE_RADIUS = NEEDLE_WIDTH / 2
+const NEEDLE_CENTER_RADIUS = NEEDLE_RADIUS / 3
+const NEEDLE_BORDER_WIDTH = WHEEL_OUTLINE_BORDER_WIDTH
+const NEEDLE_CENTER = {
+  X: WHEEL_CENTER.X,
+  Y: NEEDLE_RADIUS + MARGIN
+}
+const NEEDLE_TIP = {
+  X: WHEEL_CENTER.X,
+  Y: NEEDLE_RADIUS + MARGIN + NEEDLE_HEIGHT
+}
+
+const WHEEL_OUTLINE_COLOR = '#ff2800'
+const WHEEL_OUTLINE_BORDER_COLOR = '#d21404'
+const WHEEL_COLOR_FRONT = '#ff2800'
+const WHEEL_COLOR_BACK = '#fca311'
+const NEEDLE_COLOR = '#fff299'
+const NEEDLE_BORDER_COLOR = '#ffd901'
+
+const WHEEL_CENTER_SHADOW_BLUR = 100
+const WHEEL_CENTER_SHADOW_COLOR = '#000'
+const WHEEL_OUTLINE_SHADOW_BLUR = 20
+const WHEEL_OUTLINE_SHADOW_COLOR = '#000'
+const NEEDLE_SHADOW_BLUR = 15
+const NEEDLE_SHADOW_COLOR = '#000'
+
+export const WheelNeedle = () => (
+  <>
```

Comment:
The most top <> is not required, suggest to remove

Reply:
fix in 87db9cf